docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,56 @@
+version: "3"
```

Comment:
Can this file be moved to `uffizzi`'s directory?

Reply:
done !